### PR TITLE
Notify to change selected network

### PR DESCRIPTION
**Pull request by haythem96**
This PR related to #82 .
It check the selected network and the network Kredits contract deployed at and notify the user with a simple alert in case both networks are not the same without running `kredits.setup()`.

### CODE REVIEW

**Comment by haythem96** [comment]
I'm not really sure if this is the right way to do it with the existing project architecture...

File: app/services/kredits.js [hunk 1]
```diff
@@ -136,6 +136,10 @@ export default Service.extend({
     });
   },
 
+  getKreditsDeployedNetwork() {
+    return Kredits.availableNetworks();
```
Comment (skddc) [inline app/services/kredits.js]: I don't understand how one can get a single network from a list of networks without filtering here.

File: app/routes/application.js [hunk 2]
```diff
@@ -7,19 +7,55 @@ export default Route.extend({
...
-        console.error(error);
-      });
-      if (kredits.get('accountNeedsUnlock')) {
-        if (confirm('It looks like you have an Ethereum wallet available. Please unlock your account.')) {
-          transition.retry();
+    let selectedNetworkId = window.ethereum.networkVersion;
+    let kreditsNetwork = kredits.getKreditsDeployedNetwork();
+    let kreditsNetworkName;
+
+    switch(kreditsNetwork) {
```
Comment (skddc) [inline app/routes/application.js]: This belongs in a separate function or config. But from what I remember I'm pretty sure you can get network names from the `ether.js` library/API anyway, so there's no need to re-implement our own version of this.

File: app/services/kredits.js (hunk 1 above)
Comment (haythem96) [reply app/services/kredits.js]: mm I thought that there will be only the network where kredits is deployed at (etiher rinkeby, ropsten, mainnet...), or no ?

Comment (skddc) [reply app/services/kredits.js]: We run a local network for development, making it at least two. But also, there could be more networks in the future, hence the plural in `availableNetworks()` (as opposed to `getNetwork()` or similar).

We talked about this in the call yesterday and @bumi had some good ideas. Would be good to write them down in this PR.

Comment (haythem96) [reply app/services/kredits.js]: yeah really sorry for missing the last calls... was trying to join but I miss for some reasons... I will be available on the next ones.

**Pull request closed without merging**